pkg/oci/utils_test.go

```diff
@@ -147,7 +147,7 @@ func TestMinimalPodConfig(t *testing.T) {
 		t.Fatalf("Could not parse config.json: %v", err)
 	}
 
-	podConfig, err := PodConfig(ociSpec, runtimeConfig, tempBundlePath, containerID, consolePath)
+	podConfig, err := PodConfig(ociSpec, runtimeConfig, tempBundlePath, containerID, consolePath, false)
```

Comment:
Should there be a test that checks that the for console redirection is working as defined by the detach parameter?